app/http.php

```diff
@@ -266,7 +264,8 @@ function logError(Throwable $error, string $action, Utopia\Route $route = null):
 
         $response
             ->setStatusCode($client->getStatusCode())
-            ->json(\json_decode($client->getBody(), true));
+            ->setContentType(Response::CONTENT_TYPE_JSON, self::CHARSET_UTF8)
```

Comment:
In theory, we can accept any content type that the executor decides to return, maybe we can just set the headers and be a real proxy, not forcing anything.

Reply:
Makes sense. Removing forced JSON headers.